src/plugins/workspace/server/saved_objects/workspace_ui_settings_client_wrapper.ts

```diff
@@ -61,13 +63,24 @@ export class WorkspaceUiSettingsClientWrapper {
           options
         );
 
-        const workspaceObject = await this.getWorkspaceTypeEnabledClient(
-          wrapperOptions.request
-        ).get<WorkspaceAttribute>(WORKSPACE_TYPE, requestWorkspaceId);
+        let workspaceObject: SavedObject<WorkspaceAttribute> | null = null;
+
+        try {
+          workspaceObject = await this.getWorkspaceTypeEnabledClient(wrapperOptions.request).get<
+            WorkspaceAttribute
+          >(WORKSPACE_TYPE, requestWorkspaceId);
+        } catch (e) {
+          this.logger.error(`Unable to get workspaceObject with id: ${requestWorkspaceId}`);
+        }
+
+        configObject.attributes = {
+          ...configObject.attributes,
+          ...(workspaceObject ? workspaceObject.attributes.uiSettings : {}),
+        };
 
         configObject.attributes = {
```

Comment:
Seems the `configObject.attributes` has already been set above. Why we need to assign it again?

Reply:
You are right, remove the useless assign.